docs/ref/django-admin.txt

```diff
@@ -966,6 +966,12 @@ behavior you can use the ``--no-startup`` option. e.g.::
 
     django-admin shell --plain --no-startup
 
+.. versionadded:: 1.10
```

Comment:
Please add a `.. django-admin-option::` heading and a mention in the 1.10 release notes.